README.md

```diff
@@ -5,4 +5,22 @@ Web app for checking tmt tests, plans and stories
 2. Install the requirements
 3. Use the `start_api.sh` script to start the api
 ## Tests
-To run the tests, use the pytest command
\ No newline at end of file
+To run the tests, use the pytest command
+
+# Requirements
+## As a user I want to have a clickable link to test/plan verifying the issue
```

Comment:
Let's be more explicit and separate (and explain) the three expected use cases:

* test
* plan
* test + plan

It would be good to make it clear for each of them what they represent.

Reply:
The content has been moved to the stories, so the README adjustments can be dropped?